CovidVaccineHPI/worker.js

```diff
@@ -95,7 +95,7 @@ const getData = async () => {
     const mappedResults = { 
         meta : {
             LATEST_ADMINISTERED_DATE,
-            PUBLISHED_DATE : todayDateString()
+            PUBLISHED_DATE : new Date(todayDateString()).toISOString().split('T')[0]
```

Comment:
What's going on here?  This will not appear as a pacific time date when deployed.

Reply:
I changed this because the format of this date string was coming out mm/dd/yyyy which was breaking the schema validation, we could leave this as it was and change the schema instead

Reply:
The locale settings should take care of the formatting...
new Date().toLocaleString("en-CA", {timeZone: "America/Los_Angeles"})
Does it not show up as "2021-03-08..." on your device?